tools/functional-tester/etcd-tester/stresser.go

```diff
@@ -83,7 +90,7 @@ func (s *stresser) Stress() error {
 					Value: []byte(randStr(s.KeySize)),
 				})
 				putcancel()
-				if grpc.ErrorDesc(err) == context.Canceled.Error() {
+				if checkCancelErr(err) {
```

Comment:
how about just removing this func and return on error? also we should get rid of the fail count at line 98.